artview/components/select_region.py

```diff
@@ -173,39 +172,52 @@ def button_press_callback(self, event):
                                        marker='o')
                     self.previous_point = [x, y]
```

Comment:
We may want to decrease the marker size? I find them kind of large to work with.
And one issue I would love to make happen is to not have to "close" the polygon fully, but have an automatic closure if an action was chosen.